src/db/models/submit.rs

```diff
@@ -64,18 +64,19 @@ impl Submit {
         database_connection.transaction::<_, Error, _>(|conn| {
             diesel::insert_into(submits::table)
                 .values(&new_submit)
-
                 // required because if we re-use the staging store, we do not create a new UUID but re-use the old one
                 .on_conflict_do_nothing()
-
                 .execute(conn)
```

Comment:
Not ideal but it should still be clear that the comment belongs to the next line.